tests/marqo_test.py

```diff
@@ -128,6 +129,9 @@ def side_effect(http_operation, path, body=None, content_type=None, index_name="
                             if isinstance(response, InternalError):
                                 raise response
                             return response
+                        # performs check for get indexes which needs to pass for mappings during cloud tests
+                        if http_operation == "get" and path == "" and body is None and content_type is None:
+                            return
```

Comment:
if we don't add specific catch that triggers only on requests.get sent by mappings as far as I am concerned the next stroke with forbid_extra_calls would raise errors

Reply:
Sorry can you break explain this to me in a simpler way?

Reply:
1. We execute some test that uses mock_http_traffic decorator and has forbid_extra_calls=True with is_marqo_cloud set to True.
2. If we don’t add any line for catching /indexes request it will not match path and on the check for forbid_extra_calls on the next line it will raise error.